tokenbucket_test.go

```diff
@@ -161,3 +161,27 @@ func (s *LimitersTestSuite) TestTokenBucketOverflow() {
 		s.Equal(time.Duration(0), wait)
 	}
 }
+
+func (s *LimitersTestSuite) TestTokenBucketRefill() {
+	backend := l.NewTokenBucketInMemory()
+	clock := l.NewSystemClock()
+
+	bucket := l.NewTokenBucket(4, time.Millisecond*100, l.NewLockNoop(), backend, clock, s.logger)
+	sleepDurations := []int{150, 90, 50, 70}
+	desiredAvailable := []int64{3, 2, 2, 2}
+
+	bucket.Limit(context.Background())
+	state, err := backend.State(context.Background())
+	s.Assert().NoError(err, "unable to retrieve backend state")
```

Comment:
Since further execution does not make sense if there is an error here it should "Require()" instead.

```suggestion
	s.Require().NoError(err, "unable to retrieve backend state")
```

Reply:
done